app/build.gradle

```diff
@@ -47,11 +52,17 @@ dependencies {
     implementation 'org.jetbrains.anko:anko-common:0.10.8'
     implementation 'org.jetbrains.anko:anko-sdk15:0.10.8'
 
+    implementation "com.yubico.yubikit:yubikit:1.0.0-beta02"
```

Comment:
for QR funtionality you don't need this module and you don't need to change your namespace then...
when you're ready to switch to other yubikit functionality than you can just swap classes and remove old ones